features/local_links_manager.feature

```diff
@@ -0,0 +1,9 @@
+@app-local-links-manager
+Feature: Local Links Manager
+
+  @local_network
+  Scenario: Healthcheck
+    Given I am testing "local-links-manager" internally
+    When I request "/healthcheck"
+    Then Json is returned
```

Comment:
```suggestion
    Then JSON is returned
```

Not sure if this works either way, but [JSON would normally be all capitalised](https://github.com/alphagov/smokey/search?q=JSON+is+returned).

Reply:
👍